docs/installation.md

```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
 Below are instruction on how to build `jstz` from source. Additionally, this section
 details how to install Octez, which is used for our local sandbox.
 
-### Cloning the Repository
+### Cloning the Repositories
+
+Get the source code for both `jstz` and Octez.
 
 ```sh
 git clone https://github.com/trilitech/jstz.git
+git clone https://gitlab.com/tezos/tezos.git
 ```
 
 ### Prerequisites 📋
 
-::: tip  
-Both `jstz` and Octez are packaged with Nix, a package manager and system configuration tool that makes building from sources easy! See the [Nix docs](https://nixos.org/download.html) for instructions for your system. Additionally, ensure [Nix flakes are enabled](https://nixos.wiki/wiki/Flakes#Enable_flakes).
-:::
```

Comment:
Nix isn't necessarily a pre-requisite, but an optional one that makes things slightly easier (especially for those on MacOS). I'd prefer to keep it in the 'tips'